### PR TITLE
move to swift 5.0.1 stable release

### PR DESCRIPTION
**Pull request by satishbabariya**
moved swift version from 5.0 development snapshot to 5.0.1 release docker image.

### CODE REVIEW

File: theia-swift-docker/Dockerfile [hunk 1]
```diff
@@ -1,36 +1,15 @@
...
-
 RUN git clone --depth 1 https://github.com/apple/sourcekit-lsp
-
 WORKDIR /sourcekit-lsp/Editors/vscode
 RUN npm install
 RUN npm run postinstall
 RUN vsce package -o ./sourcekit-lsp.vsix
 
 
-FROM satishbabariya/swift as lsp-builder
```
Comment (marcdumais-work) [inline theia-swift-docker/Dockerfile]: I am guessing that the equivalent of this removed block is covered by the satishbabariya/sourcekit-lsp image, used below?

File: theia-swift-docker/Dockerfile [hunk 2]
```diff
@@ -98,4 +59,4 @@ ENV SHELL /bin/bash
 ENV USE_LOCAL_GIT true
```
Comment (marcdumais-work) [inline theia-swift-docker/Dockerfile]: I think this is no longer required - we always use the local git now.

File: theia-swift-docker/Dockerfile (hunk 1 above)
Comment (satishbabariya) [reply theia-swift-docker/Dockerfile]: yeah, but the satishbabariya/swift image is a nightly snapshot of the swift development branch, and satishbabariya/sourcekit-lsp is based on newly released swift 5.0.1

File: theia-swift-docker/Dockerfile (hunk 2 above)
Comment (satishbabariya) [reply theia-swift-docker/Dockerfile]: @marcdumais-work sure

File: theia-swift-docker/Dockerfile (hunk 1 above)
Comment (satishbabariya) [reply theia-swift-docker/Dockerfile]: in future, extension-builder will also go away after vscode extension is released

**Comment by satishbabariya** [comment]
@marcdumais-work , np, I have checked and it's working fine.

**Pull request merged**